### DISCUSSION TITLE
Is the backend open source?

### DISCUSSION THREAD

**Discussion by indicozy** [liveblocks/liveblocks#570] [Help] [answered]
Hi there! Is the server side of the codebase open source?

**Comment by GuillaumeSalles** [comment, marked as answer]
Hi @indicozy,

The backend is not open source. It might at some point, but not in the short term.

**Reply by devshahofficial** [reply]
@GuillaumeSalles hey, Any plans to open source the backend? Really love building on LiveBlocks!

**Comment by nyxb** [comment]
But it was said in another issue that it will soon be made open source. It would be really great if opensource is opensource and you could host it yourself. #1466

**Comment by N0N1m3** [comment]
@GuillaumeSalles @nvie Any updates on this?
